.github/labeler.yml

```diff
@@ -1,48 +1,63 @@
 # Add 'Multicast' label to any root file changes
 Multicast:
-  - setup.py
-  - multicast/*.py
+  - changed-files:
+    - any-glob-to-any-file:
+      - setup.py
+      - multicast/*.py
+      - LICENSE
+      - LICENSE.md
 
 # Add 'Lint' label to any lint file changes
 Linter:
-  - .stickler.yml
-  - .hound.yml
-  - .yamllint.conf
-  - .flake8.ini
-  - .deepsource.toml
-  - tests/check_*
+  - changed-files:
+    - any-glob-to-any-file:
+      - .stickler.yml
+      - .hound.yml
+      - .yamllint.conf
+      - .flake8.ini
+      - .deepsource.toml
+      - tests/check_*
 
 # Add 'Lang' label to any python file changes
 Python Lang:
-  - **/*.py
-  - *.py
+  - changed-files:
+    - any-glob-to-any-file:
+      - '**/*.py'
+      - '*.py'
```

Comment:
This should fix the labeler once merged (will remain broken until in master branch unfortunately)